specification/sbi_atee.adoc

```diff
@@ -1437,63 +1533,104 @@ Both `cert_request_addr`, `request_data_addr` and `cert_addr_out` must be 4kB-al
+process where a set of TEE pages are assigned for a TVM to hold a TVM’s
+global state. This routine also configures the global configuration that
+applies to the TVM and affects all TVM hart settings. For example, features
+enabled for this TVM, perfmon enabled, debug enabled
+etc.
+
+| <<sbi_tee_host_finalize_tvm, sbi_tee_host_finalize_tvm>> | This operation enables a
+VMM to finalize the measurement of a TVM (static). The TSM enforces that a
+TVM virtual harts cannot be entered unless the TVM measurement is committed
+via this operation.
+
+| <<sbi_tee_host_destroy_tvm, sbi_tee_host_destroy_tvm>> | TVM shutdown verifies VMM
+has stopped all virtual hart execution for the TVM. The TVM virtual hart
+may not be entered after this point. The VMM may start reclaiming TVM
+memory after this point.
+
+| <<sbi_tee_host_add_tvm_memory_region, sbi_tee_host_add_tvm_memory_region>> | Adds a
+memory region to the TVM at the specified range of guest physical address space. The
+memory range is confidential to the guest and may only be populated with confidential
+pages.
 
 | <<sbi_tee_host_add_tvm_page_table_pages, sbi_tee_host_add_tvm_page_table_pages>> | Add 
 one or more page mappings to the G-stage translation structure for a TVM.
 The pages to be used for the G-stage page table structures must have been converted
 (and tracked) by the TSM as TEE pages; otherwise this operation will not succeed.                                                                     
 
-| <<sbi_tee_host_add_tvm_measured_pages, sbi_tee_host_add_tvm_measured_pages>> | Add 
-a page for an existing mapping for a TVM page - this add_pre must be performed before 
-finalization of the TVM measurement via sbi_tee_host_finalize_tvm. For this 
-operation, the VMM must provide the page contents that get copied into 
-confidential memory pages for the TVM (and get tracked, encrypted etc). The 
-contents of these pages are also measured via the sbi_tee_host_add_tvm_measured_pages, 
-including the GPA at which the page is mapped. After the TVM msmt is 
-finalized via sbi_tee_host_finalize_tvm, no more pre-add pages are allowed by 
-the TSM for that TVM.
+| <<sbi_tee_host_add_tvm_measured_pages, sbi_tee_host_add_tvm_measured_pages>> | Copies the
+given number of pages from non-confidential memory at `source_address` to confidential
+memory at `dest_addr`, then measures and maps the pages at `dest_addr` at the TVM physical
+address space at `tvm_guest_gpa`. The mapping must lie within a region of confidential memory
+created with `sbi_tee_host_add_tvm_memory_region()`. This call must not be made after calling
+`sbi_tee_host_finalize_tvm()`.
+
+This operation is used to extend the static measurement for a TVM for added page contents.
+The operation performs a SHA384 hash extend to the measurement register managed 
+by the TSM on a 256 byte block of the page. The page must be added to a 
+valid GPA mapping via the add_pre_init operation. The GPA of the page 
```

Comment:
remove the text "The page must be added to a valid GPA mapping via the add_pre_init operation."